src/Middleware/Spa/SpaProxy/src/SpaProxyLaunchManager.cs

```diff
@@ -42,7 +42,10 @@ public void StartInBackground(CancellationToken cancellationToken)
         {
             if (_launchTask == null)
             {
-                _logger.LogInformation($"No SPA development server running at {_options.ServerUrl} found.");
+                if (_logger.IsEnabled(LogLevel.Information))
+                {
+                    _logger.LogInformation($"No SPA development server running at {_options.ServerUrl} found.");
```

Comment:
```suggestion
                    _logger.LogInformation("No SPA development server running at {serverUrl} found.", _options.ServerUrl);
```
And the rest of this PR.